test/e2e/egressqos.go

```diff
@@ -0,0 +1,212 @@
+package e2e
```

Comment:
you want to go ahead and drop this commit and then make a new PR for it later?

Reply:
dropped